docs/requirements.txt

```
@@ -5,7 +5,7 @@ colorama==0.4.6
 ghp-import==2.1.0
 idna==3.4
 Jinja2==3.1.2
-Markdown==3.4.3
+Markdown==3.3
```

We probably want the patch version here.

```suggestion
Markdown==3.3.7
```